simulators/ethereum/pyspec/go.mod

```diff
@@ -50,10 +55,14 @@ require (
 	github.com/tklauser/go-sysconf v0.3.11 // indirect
 	github.com/tklauser/numcpus v0.6.0 // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
-	golang.org/x/crypto v0.6.0 // indirect
-	golang.org/x/exp v0.0.0-20230206171751-46f607a40771 // indirect
-	golang.org/x/sys v0.5.0 // indirect
-	golang.org/x/text v0.7.0 // indirect
+	golang.org/x/crypto v0.9.0 // indirect
+	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc // indirect
+	golang.org/x/sync v0.2.0 // indirect
+	golang.org/x/sys v0.8.0 // indirect
+	golang.org/x/text v0.9.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
+	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/ethereum/go-ethereum v1.12.0 => github.com/marioevz/go-ethereum v1.10.14-0.20230607235252-075f62d72ccc
```

Comment:
Why is this replacement needed?

Reply:
Currently its the only branch (I believe) with the dataGasUsed and excessDataGas fields added the `*core.Genesis` struct which we are using in `types.go`

This is the required type here: https://github.com/ethereum/hive/blob/master/simulators/ethereum/engine/client/hive_rpc/hive_rpc.go#L40
that we use to start a clients with a genesis file alongside the EngineAPI

Maybe there is a better way to do this, that allows us to use a custom genesis struct?

Nonetheless once 4844 is merged no replacement will be required!

Reply:
I see. Please please, if you need certain fields added in go-ethereum, send a PR to go-ethereum!